agent/proxycfg/testing_ingress_gateway.go

```diff
@@ -1364,7 +1364,7 @@ func TestConfigSnapshotIngressGateway_TLSMinVersionListenersGatewayDefaults(t te
 					Port:     8083,
 					Protocol: "http",
 					Services: []structs.IngressService{
-						{Name: "s3"},
+						{Name: "s4"},
```

Comment:
This was misconfigured originally but was only run as a cluster test so it failed when trying to generate listeners.